__tests__/integration.test.js

```diff
@@ -7,21 +7,44 @@ const app = require("../app/app");
 beforeAll(() => seed(data));
 afterAll(() => db.end());
 
-describe("GET /api/treasures", () => {
-    test("200: returns an array of the topic objects", () => {
+describe("GET /api/topics", () => {
+  test("200: returns an array of the topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        for (const topic of topics) {
```

Comment:
if you are going to make any assertions within a loop,  please do a length check.  i think and empty array would lead to a false pass.